code/src/components/Action.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react'
+import { useSelector, useDispatch } from 'react-redux'
+import { executeAction } from '../reducers/game'
+import styled from 'styled-components'
+
+
+const MoveButton = styled.button`
+  background: black;
+  color: #4CFF42;
+  font-family: 'Inconsolata', monospace;
+  font-size: 15px;
+  padding: 10px;
+`
+
+export const Action = ({ props }) => {
+
+  const dispatch = useDispatch();
+  const [description] = useState(props.description)
+  const [direction] = useState(props.direction)
+  const [type] = useState(props.type)
+  const username = useSelector((store) => store.game.username);
+
+  const handleOnClick = () => {
+    dispatch(executeAction(username, type, direction))
+  }
+
+  return (
+    <div>
+      <h3>Go {direction} </h3>
+      <p>{description}</p>
+      <p></p>
+      <MoveButton onClick={handleOnClick}>Proceed</MoveButton>
+    </div>
```

Comment:
From an UX and accessibility perspective we would prefer the button to have the {direction}. So the buttons say either "Go west" or "Go north" for example. We don't think it's a good idea to have two buttons with the same text.